terraform/032-db-backup/task-definition.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "volumesFrom": [],
+    "memory": ${memory},
+    "extraHosts": null,
+    "dnsServers": null,
+    "disableNetworking": null,
+    "dnsSearchDomains": null,
+    "portMappings": [],
+    "hostname": null,
+    "essential": true,
+    "entryPoint": null,
+    "mountPoints": [],
+    "name": "cron",
```

Comment:
Should `cron` here instead be something like `backup` or `db-backup`?

Reply:
eh, doesnt matter, it runs cron